### PR TITLE
Dk 001 test global tool configuration page

### PR DESCRIPTION
**Pull request by dsk1975**

### CODE REVIEW

**Review by maximashuev — COMMENTED** [review]
Looks. good.
Only one thing, naming convention. It would be more readable if it looks like this
locators_global_tool_config_page.py 
test_global_tool_config_page.py

**Comment by maximashuev** [comment]
Please add a description with the links to Trello tasks like here 
https://github.com/Python2-QAForEveryone/jenkins-testing/pull/36

**Comment by dsk1975** [comment]
> Looks. good.
> Only one thing, naming convention. It would be more readable if it looks like this
> locators_global_tool_config_page.py
> test_global_tool_config_page.py

Thank you, I will rename it.

**Comment by dsk1975** [comment]
> Please add a description with the links to Trello tasks like here
> #36

For sure.

File: tests/locators_global_tool_config_page.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+from selenium.webdriver.common.by import By
```
Comment (Wis13) [inline tests/locators_global_tool_config_page.py:1]: Этот файл нужно удалить, а локаторы перенести в GlobalTollConfiguration.py

File: pages/GlobalToolConfiguration.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from pages.BasePage import BasePage
```
Comment (Wis13) [inline pages/GlobalToolConfiguration.py:1]: Вроде договорились что все наследники от BasePage  заканчиваются в названии Page.

GlobalTollConfiguration.py -------> GlobalTollConfigurationPage.py

File: tests/locators_global_tool_config_page.py (hunk 1 above)
Comment (maximashuev) [reply tests/locators_global_tool_config_page.py:1]: Yes, please. Remove this file and move all locators to your page model.
example [link](https://github.com/Python2-QAForEveryone/jenkins-testing/pull/41)

**Review by maximashuev — CHANGES_REQUESTED** [review]

**Comment by dsk1975** [comment]
Closed this PR because I got a lot of conflicts. Going to build from scratch and open a new PR taking into the previous comments.

**Pull request closed without merging**